examples/example_mod.m

```diff
@@ -0,0 +1,23 @@
+% Initialization.
+clear;
+setup;
+n = neuron.Neuron();
+
+% Try to find 'hd' mechanism.
+axon1 = n.Section("axon1");
+axon1.insert_mechanism("hd");  % Warning: Insertable mechanism 'hd' not found. 
```

Comment:
Shouldn't this be an error not a warning? If it's a warning, the simulation will still run and generate the wrong results (since it will be missing an ion channel).

Reply:
Maybe we need to add an issue for looking again at all warnings/errors. I think right now almost everything is a warning, which is sometimes easier during development.

Reply:
So for what it's worth, in NEURON's CI, we have tests that are expected to generate an error and we consider it a test failure if they do not.